AcademyChallenge/Data/Network/DTO/EmojiResponse.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  EmojiResponse.swift
+//  AcademyChallenge
+//
+//  Created by Keven Esmael on 04/10/2022.
+//
+
+import Foundation
+import CoreData
+
+
+struct EmojiResponse: Decodable {
+	
+	var emojis: [Emoji] = []
+	let persistence:EmojiCoreData = EmojiCoreData()
```

Comment:
The CoreData object should not be associated with the response object directly, that would make it harder to test this object in the future. Ideally this should be moved to the file that works as the data source of this process (a Service / ViewModel class)